File: src/main/java/co/gatedaccess/web/http/TokenBody.java
package co.gatedaccess.web.http;

public class TokenBody {
    String token;

    public TokenBody() {
    }

    public TokenBody(String token) {
        this.token = token;
    }


    public String getToken() {
        return token;
    }

    public TokenBody setToken(String token) {
        this.token = token;
        return this;
    }
}

File: src/main/kotlin/co/gatedaccess/web/http/response/TokenBody.kt
package co.gatedaccess.web.http.response

class TokenBody {
    var token: String? = null

    constructor()

    constructor(token: String?) {
        this.token = token
    }


    fun setToken(token: String?): TokenBody {
        this.token = token
        return this
    }
}
